packages/adapters/subgraph/src/reader.ts

```diff
@@ -171,23 +177,31 @@ export class SubgraphReader {
       [...txIdsByDestinationDomain.entries()].map(async ([destinationDomain, transferIds]) => {
         const subgraph = this.subgraphs.get(destinationDomain)!; // should exist bc of initial filter
 
-        await subgraph.runtime.request<GetExecutedAndReconciledTransfersByIdsQuery>(
+        const { transfers } = await subgraph.runtime.request<GetExecutedAndReconciledTransfersByIdsQuery>(
           (client) =>
             client.GetExecutedAndReconciledTransfersByIds({
               transferIds,
-              maxXCalledBlockNumber: agents.get(destinationDomain)!.maxXCallBlockNumber.toString(),
+              maxXCalledBlockNumber: agents.get(destinationDomain)!.maxBlockNumber.toString(),
```

Comment:
As far as I can tell, we are no longer using this param in `GetExecutedAndReconciledTransfersByIds`? It looks like it's been removed from the query

Reply:
Although, we might want to add it back in? Unsure